components/cli/pkg/commands/list_images.go

```diff
@@ -39,9 +39,18 @@ type Component struct {
 	ports               []int
 	deployment, service string
 }
+type ImageData struct {
```

Comment:
Do we need to access this struct from other packages?

Reply:
No.

Reply:
Then please change it to package private.

Reply:
Updated.